man/qmethod.Rd

```diff
@@ -44,9 +74,9 @@ Returns a list of class \code{QmethodRes}, with eight objects:
 See further references on the methodology in \code{\link{qmethod-package}}.}
 
 \note{
-In Q methodology there are commonly two techniques to extract the factors: principal components analysis (PCA) and centroid factor analysis. 
-
-This package uses PCA because this function is already implemented in R. If you find a function in R that suitably implements centroid factor analysis as used in Q method (See Brown, 1980), please report to the author of this package: \email{aiora.zabala@gmail.com}.
+  In Q methodology principal components analysis (PCA) and centroid factor analysis (CFA) (see Brown, 1980) are commonly used extraction techniques, as well as, recently, principal axis factoring (PAF).
```

Comment:
PAF is commonly used? I'm not sure about that, please eliminate. Lets stick to the bare minimum.